include/yaml-cpp/scanner.h

```diff
@@ -15,9 +15,9 @@
 #include <stack>
 #include <string>
 
-#include "ptr_vector.h"
-#include "stream.h"
-#include "token.h"
+#include "yaml-cpp/ptr_vector.h"
```

Comment:
Why these changes?

Reply:
consistency

Reply:
These files aren't in the set of public headers; they're just in the src directory. That's why they're included as relative names. I'm surprised it still compiles - that's a bug. In any case, please revert these and just leave the additions you need to make msvc happy.

Reply:
These files (ptr_vector/etc) have been moved to the include directory as they're required by scanner.h (at least)

so I can replace 

```#include "yaml-cpp/ptr_vector.h"``` with ```#include "yaml-cpp/ptr_vector.h"``` for current directory lookup but I don't see the difference, with the first you know that it's in the include directory

Reply:
Oh, wow, I totally missed that you moved these files to the public directory.

Hm. I'd prefer if we didn't extend the public API here. It looks like the error is just for the move constructor and assignment, right? If so, I don't think the Parser actually needs to be moveable, so you could just change those to be = delete instead of = default.

If we actually do need to move these, then you can just move the definition of the move constructor/assignment to the src file (still writing = default).